### PR TITLE
fix: update the foas staging release to use `stage` instead of staging when using `foascli split`

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes
This PR fixes an issue with the staging release process where we are using `staging` instead of `stage` when running `foascli split` command

### CODE REVIEW

File: .github/scripts/split_spec.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+
+env_flag=${target_env:?}
```
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply .github/scripts/split_spec.sh:5]: from the action that is using the script

>         - name: Run foascli split command
>           id: split
>           env:
>             target_env: "prod"
>           run: .github/scripts/split_spec.sh

[the remark this answers is not in the document]
Comment (blva) [reply .github/scripts/split_spec.sh:5]: and why can't we update it there with "stage"?

**Review by blva — APPROVED** [review]

File: .github/scripts/split_spec.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply .github/scripts/split_spec.sh:5]: good point let me update it

[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply .github/scripts/split_spec.sh]: updated. Thanks for the feedback

**Pull request merged**